src/components/LoadedTable.res

```diff
@@ -906,20 +922,6 @@ let make = (
       </DesktopView>
     </div>
 
-  let customizeColumsButtons = {
-    switch clearFormattedDataButton {
-    | Some(clearFormattedDataButton) =>
-      <div className={`flex flex-row mobile:gap-7 desktop:gap-10 ${filterBottomPadding}`}>
-        clearFormattedDataButton
-        <Portal to={""}> rightTitleElement </Portal>
```

Comment:
why is this removed

Reply:
Portal is not required here as {to} is empty string, it is simply rendering the children.